### ISSUE TITLE
creating tasks with "id" in their name leads to an error

### ISSUE THREAD

**Issue by djugei** [tbabej/taskwiki#239]
there is probably some escaping missing.

**Comment by lucasdf** [comment]
@djugei working fine here with `test: check user id`. Do you have some example?

**Comment by djugei** [comment]
yeah creating a task called id_test crashes, maybe the id needs to be in the front

**Comment by nkakouros** [comment]
I cannot reproduce this. @djugei Is this still the case?

**Comment by tbabej** [comment]
I think resolution of this issue would require us getting more information about the environment, in particular the Taskwarrior version. This used to be an issue directly with TW in the past.

**Comment by djugei** [comment]
i could reproduce this on an arch, i think i build the stuff from aur, but i am not actively using tw or taskwiki anymore. id needed to be in the front, not just anywhere.

**Comment by lanker** [comment]
Seems to be an issue in taskwarrior: https://github.com/GothenburgBitFactory/taskwarrior/issues/1830, https://github.com/GothenburgBitFactory/taskwarrior/issues/1908, etc. Supposed to be fixed in the 2.6.0 branch.
